### PR TITLE
New version: Kitware.CMake version 3.23.0-rc1

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/45416)

### CODE REVIEW

**Review by KaranKad — CHANGES_REQUESTED** [review]
This is a release client version but with no way do differentiate it from stable release, so it might be better to not add it.

**Comment by vedantmgoyal2009** [comment]
I don't know much about release management, but according to this, I think we can add it because "this release goes into production". 
image

File: manifests/k/Kitware/CMake/3.23.0-rc1/Kitware.CMake.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.0.6 using InputObject 🤖 $debug=QUSU-7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: Kitware.CMake
+PackageVersion: 3.23.0-rc1
```
Comment (KaranKad) [inline manifests/k/Kitware/CMake/3.23.0-rc1/Kitware.CMake.yaml:5]:
```suggestion
PackageVersion: 3.23.0
```

File: manifests/k/Kitware/CMake/3.23.0-rc1/Kitware.CMake.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# Created with YamlCreate.ps1 v2.0.6 using InputObject 🤖 $debug=QUSU-7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Kitware.CMake
+PackageVersion: 3.23.0-rc1
```
Comment (KaranKad) [inline manifests/k/Kitware/CMake/3.23.0-rc1/Kitware.CMake.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 3.23.0
```

File: manifests/k/Kitware/CMake/3.23.0-rc1/Kitware.CMake.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# Created with YamlCreate.ps1 v2.0.6 using InputObject 🤖 $debug=QUSU-7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Kitware.CMake
+PackageVersion: 3.23.0-rc1
```
Comment (KaranKad) [inline manifests/k/Kitware/CMake/3.23.0-rc1/Kitware.CMake.installer.yaml:5]:
```suggestion
PackageVersion: 3.23.0
```

**Review by KaranKad — CHANGES_REQUESTED** [review]

File: manifests/k/Kitware/CMake/3.23.0-rc1/Kitware.CMake.yaml (hunk 1 above)
Comment (KaranKad) [reply manifests/k/Kitware/CMake/3.23.0-rc1/Kitware.CMake.yaml:5]: image

**Review by ItzLevvie — APPROVED** [review]

**Comment by Carbenium** [comment]
@vedantmgoyal2009 @KaranKad @ItzLevvie Please consider not adding RCs (Release Candidates) to winget without an explicit opt-in. The default should be stable releases only.
The citation given by @vedantmgoyal2009 (from https://en.wikipedia.org/wiki/Software_release_life_cycle) is not really applicable. Yes, it might be true that the last RC before stable is promoted to the stable version, but that does not mean that RC1 (the first RC) is actually going stable (as seen with 3.23.0-rc2).

**Pull request merged**